models/models_test.go

```diff
@@ -106,10 +106,11 @@ func TestModels(t *testing.T) {
 		require.NoError(t, err)
 		expected := []*models.Node{
 			{
-				NodeID:    "N1",
-				NodeType:  models.GenericNodeType,
-				NodeName:  "N1 name",
-				CreatedAt: now,
+				NodeID:       "N1",
+				NodeType:     models.GenericNodeType,
+				NodeName:     "N1 name",
+				CreatedAt:    now,
+				CustomLabels: []byte{},
```

Comment:
If `nil` CustomLabels doesn't work, that is a bad regression.  We should do something about. For example, set it in `BeforeInsert`/`BeforeUpdate`.